wgpu-types/src/lib.rs

```diff
@@ -730,6 +730,15 @@ bitflags::bitflags! {
         /// This is a native only feature.
         const VERTEX_ATTRIBUTE_64BIT = 1 << 53;
 
+        /// Allows vertex shaders to have outputs which are not consumed
+        /// by the fragment shader.
+        ///
+        /// Supported platforms:
+        /// - Vulkan
+        /// - Metal
+        /// - OpenGL
+        const SHADER_UNUSED_VERTEX_OUTPUT = 1 << 54;
+
         // 54..59 available
```

Comment:
nit: 55..59

Reply:
Yeah, this was missed in the original as well: https://github.com/gfx-rs/wgpu/pull/4116#pullrequestreview-1667129478

Reply:
Either way, fixed here!